coremltools/converters/mil/frontend/torch/internal_graph.py

```diff
@@ -439,12 +441,23 @@ def from_exir(cls, exir):
 
             params[name if name not in parameters_to_inputs else parameters_to_inputs[name]] = value
 
-        graph = exported_program.graph
+        graph_module = exported_program.graph_module
+        graph = graph_module.graph
 
         outputs = []
-        for node in graph.nodes:
+        for node_without_typing in graph.nodes:
+            node: torch.fx.Node = node_without_typing
             if node.op == "call_function":
                 nodes.append(InternalTorchIRNode.from_exir_node(node=node))
+            elif node.op == "get_attr":
```

Comment:
There's shouldn't be any `get_attr` nodes unless `control flow` is involved if recall the contract correctly.

Reply:
Yeah unfortunately we will be seeing some `get_attr` nodes getting constants, due to an issue found in [lifting constants to buffer](https://github.com/pytorch/pytorch/issues/110484)